### ISSUE TITLE
[a11y-defect-2]: Request an extension error's ID does not match aria-describedby's ID

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#63226]
## Point of contact

joshkimux

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

<img width="1408" alt="Screenshot 2023-08-07 at 1 54 18 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/7158af78-d65a-4b07-8b51-d845d445e6cd">

Currently on the reason for extension page:
- the error message's ID is `input-error-message`
- the text area's aria-describedby attribute is `error-message`

As a result, the user won't hear the error message when shift-tabbing back to the field.

## Reproduction steps

1. Log in as user 228
2. Go to the reason for extension page
3. Turn on JAWs (or inspect)
4. Skip the reason for extension page
5. Shift tab back to the extension text area
6. Confirm that the error message "this field cannot be left blank" is not read out

### Acceptance criteria
- [ ] The error message's ID is `error-message`

### Recommendation
The design system component uses `error-message` as the ID for the text area's error message, which connects to the text-area's aria-describedby attribute.

<img width="1308" alt="Screenshot 2023-08-07 at 2 00 34 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/cd2f7dbc-6215-464c-b26e-c447e463e713">

## Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by data-doge** [comment]
Note from bug triage: Waiting on merging in component library update in vets-website. @Mottie

**Comment by Mottie** [comment]
The Design System Team releases new updates at the end of their sprint (every 2 weeks), which just happened on 8/17. So the next update will occur around 8/31. We can test this change after that.

**Comment by HeatherWidmont** [comment]
@joshkimux can you please take a look at this ticket & close it if it meets requirements?

**Comment by joshkimux** [comment]
Will give this a look!

**Comment by joshkimux** [comment]
Closing!
<img width="1460" alt="Screenshot 2023-08-30 at 4 37 49 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/a7cbf349-e811-49b0-aa9a-984a8a886656">
